microraiden/microraiden/client/default_http_client.py

```diff
@@ -15,42 +17,56 @@ class DefaultHTTPClient(HTTPClient):
     def __init__(
             self,
             client: Client,
-            api_endpoint: str,
-            api_port: int,
             retry_interval: float = 5,
             initial_deposit: Callable[[int], int] = lambda price: 10 * price,
             topup_deposit: Callable[[int], int] = lambda price: 5 * price
     ) -> None:
-        super().__init__(client, api_endpoint, api_port)
+        HTTPClient.__init__(self, client)
```

Comment:
Isn't `super().__init__` preferrable in P3?

Reply:
As far as I know, for single inheritance explicit calls are preferred. It's also more informative in my opinion. But I don't really have a preference.

Reply:
I am also ok with either, both are ugly IMO :) But that's Python.